.github/workflows/e2e-test-darts-cifar10.yaml

```diff
@@ -33,6 +33,6 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetes-version: ["v1.27.11", "v1.28.7", "v1.29.2"]
+        kubernetes-version: ["v1.28.7", "v1.29.2", "v1.30.7"]
```

Comment:
We can also add one that tests whatever is the cutting edge version so we can get notified of any incompatibility earlier

Reply:
SGTM. WDYT👀 @kubeflow/wg-automl-leads